content/authors/james-munroe/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+# Display name
+title: "James Munroe"
+
+# Username (this should match the folder name and the name on publications)
+authors:
+- "james-munroe"
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position (e.g., Professor of Artificial Intelligence)
+role: Product and Community Lead
+
+# Organizations/Affiliations
+organizations:
+- name: 2i2c
+  url: "https://2i2c.org"
+
+# Short bio (displayed in user profile at end of posts)
+bio:
+
+# List each interest with a dash
+interests: []
+
+# Social/Academic Networking
+# For available icons, see: https://sourcethemes.com/academic/docs/page-builder/#icons
+#   For an email link, use "fas" icon pack, "envelope" icon, and a link in the
+#   form "mailto:your-email@example.com" or "#contact" for contact widget.
+social:
+- icon: envelope
+  icon_pack: fas
+  link: 'mailto:jmunroe@2i2c.org'
+- icon: twitter
+  icon_pack: fab
+  link: https://twitter.com/JamesMunroeNL
+- icon: github
+  icon_pack: fab
+  link: https://github.com/jmunroe
+
+# Enter email to display Gravatar (if Gravatar enabled in Config)
+# email: ""
+
+# Organizational groups that you belong to (for People widget)
+#   Set this to `[]` or comment out if you are not using People widget.
+user_groups:
+- Community Guidance and Partnerships Team
+---
+
+TODO: @James can you suggest an edit here?
```

Comment:
I'll probably revisit it in a few months, but for now how about this for some place holder text?

`James is Product and Community Lead for 2i2c. Coming from a background as an associate professor in the Department of Physics and Physical Oceanography at [Memorial University of Newfoundland](https://mun.ca), he is a strong advocate for enabling scientists and students to be efficient and effective in their computational workflows. Building on previous work in big data oceanography with links to the [Pangeo](https://pangeo.io/) project, [COSIMA](http://cosima.org.au/): Consortium for Ocean-Sea Ice Modelling in Australia, and [CIOOS](https://cioos.ca/): Canadian Integrated Ocean Observing System, James wants to bring the strength of the Jupyter ecosystem to users across a broad range of educational and research domains. `